hipBLAS build git checkout b2110

hello, 

cant build like the follow with latest version, does anyone knows why? 

```
cmake -H. -Bbuild \
-DLLAMA_HIPBLAS=ON \
-DAMDGPU_TARGETS="gfx803;gfx900;gfx906;gfx908;gfx90a;gfx1010;gfx1030" \
-DCMAKE_BUILD_TYPE=Release \
-DROCM_PATH=/opt/rocm \
-DCMAKE_C_COMPILER=/usr/bin/clang-15 \
-DCMAKE_CXX_COMPILER=/usr/bin/clang++-15

make -j16 -C build 
```

that makes me use all my AMD gpus all togheter

here the logs when running the build on the latest llamacpp version

```
make[2]: Leaving directory '/llama.cpp/build'
[  6%] Built target ggml
make[2]: Entering directory '/llama.cpp/build'
make[2]: Entering directory '/llama.cpp/build'
make[2]: Entering directory '/llama.cpp/build'
make[2]: Leaving directory '/llama.cpp/build'
make[2]: Leaving directory '/llama.cpp/build'
make[2]: Leaving directory '/llama.cpp/build'
make[2]: Entering directory '/llama.cpp/build'
make[2]: Entering directory '/llama.cpp/build'
make[2]: Entering directory '/llama.cpp/build'
[  7%] Building CXX object CMakeFiles/llama.dir/llama.cpp.o
[  7%] Linking CXX static library libggml_static.a
[  7%] Building CXX object examples/gguf/CMakeFiles/gguf.dir/gguf.cpp.o
[  8%] Building CXX object CMakeFiles/llama.dir/unicode.cpp.o
make[2]: Leaving directory '/llama.cpp/build'
[  8%] Built target ggml_static
[  9%] Linking CXX executable ../../bin/gguf
/usr/bin/ld: ../../CMakeFiles/ggml.dir/ggml-cuda.cu.o: in function `ggml_backend_cuda_event_wait(ggml_backend*, ggml_backend_event*)':
ggml-cuda.cu:(.text+0x360b9): undefined reference to `hipLaunchHostFunc'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [examples/gguf/CMakeFiles/gguf.dir/build.make:111: bin/gguf] Error 1
make[2]: Leaving directory '/llama.cpp/build'
make[1]: *** [CMakeFiles/Makefile2:2827: examples/gguf/CMakeFiles/gguf.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 10%] Linking CXX static library libllama.a
make[2]: Leaving directory '/llama.cpp/build'
[ 10%] Built target llama
make[1]: Leaving directory '/llama.cpp/build'
make: *** [Makefile:146: all] Error 2
make: Leaving directory '/llama.cpp/build'
```
It doesnt move from 10%, but if i do the same on version git checkout b2110 can compile without problem

--- a/ggml-cuda.cu
+++ b/ggml-cuda.cu
@@ -11541,6 +11541,7 @@ static void ggml_backend_cuda_event_wait(ggml_backend_t backend, ggml_backend_ev
     if (ggml_backend_is_cuda(event->backend)) {
         CUDA_CHECK(cudaStreamWaitEvent(g_cudaStreams[cuda_ctx->device][0], (cudaEvent_t)event->context, 0));
     } else {
+#if 0
         // untested
         auto wait_fn = [](void * user_data) {
             ggml_backend_event_t event = (ggml_backend_event_t)user_data;
@@ -11548,6 +11549,8 @@ static void ggml_backend_cuda_event_wait(ggml_backend_t backend, ggml_backend_ev
         };
 
         CUDA_CHECK(cudaLaunchHostFunc(g_cudaStreams[cuda_ctx->device][0], wait_fn, event));
+#endif
+        GGML_ASSERT(false);
     }
 }
 

